### PR TITLE
VRx, RSSI threshold from settings

### PR DESCRIPTION
**Pull request by LuisDiazUgena**

### CODE REVIEW

File: ESP32LapTimer/OLED.ino [hunk 1]
```diff
@@ -107,14 +107,17 @@ void oledUpdate(void)
...
+      display.setTextAlignment(TEXT_ALIGN_RIGHT);
+      display.drawString(127, 0, String(VbatReadingFloat, 2) + "V");
+  }
 
   // Rx modules
   display.setTextAlignment(TEXT_ALIGN_LEFT);
   for (int i = 0; i < NumRecievers; i++) {
     display.drawString(0, 9 + i * 9, getBandLabel(RXBand[i]) + String(RXChannel[i] + 1) + ", " + String(ADCvalues[i] / 12));
     display.drawProgressBar(40, 10 + i * 9, 127 - 42, 8, map(ADCvalues[i], 100, 4096, 0, 85));
+    display.drawVerticalLine(40 + map(RSSIthresholds[i], 100, 4096, 0, 85),  10 + i * 9, 8); // line to show the RSSIthresholds
```
Comment (JyeSmith) [inline ESP32LapTimer/OLED.ino:120]: Great idea!

File: README.md [hunk 2]
```diff
@@ -44,3 +44,8 @@ Compiling the Project:
 ~~Due to the fact that both the Bluetooth and Wifi stack are used quite alot of program memory is required. To compile the project you must choose 'Partition Scheme' -> Minimal SPIFFS in the Arduino IDE. Board should be selected as 'ESP32 Dev Module' in most cases.~~
 
 As we are not supoorting bluetooth for now and are using the SPIFFS partition leave the Partition Scheme as 'default'
+
+Library requirements:
+-----
+Adafruit_INA219
```
Comment (JyeSmith) [inline README.md:50]:
```suggestion
Adafruit_INA219 https://github.com/adafruit/Adafruit_INA219
```

File: README.md [hunk 3]
```diff
@@ -44,3 +44,8 @@ Compiling the Project:
...
+ESP8266 AND ESP32 OLED DRIVER FOR SSD1306 DISPLAY
```
Comment (JyeSmith) [inline README.md:51]:
```suggestion
ESP8266 AND ESP32 OLED DRIVER FOR SSD1306 DISPLAY https://github.com/ThingPulse/esp8266-oled-ssd1306
```

**Comment by AlessandroAU** [comment]
Just tested seems to be working fine. 

The mapping of RSSI thresholds on the web interface is different from the values in the app. I think we should change it to be the same.

**Pull request merged**